controllers/errors.go

```diff
@@ -0,0 +1,10 @@
+package controllers
+
+import "errors"
+
+var (
+	TargetGroupNameErr = errors.New("wrong group name")
```

Comment:
Error message is generic among all groups, so why is the error variable specific to target groups?